agent/grpc-external/services/resource/watch_test.go

```diff
@@ -129,9 +130,10 @@ func TestWatchList_TypeNotFound(t *testing.T) {
 func TestWatchList_GroupVersionMatches(t *testing.T) {
```

Comment:
I had to update these tests, so I switched to non-deprecated test helpers while I was in here.